uniqueWordAbbreviation.py

```diff
@@ -5,8 +5,8 @@ def __init__(self, dictionary):
         :type dictionary: List[str]
         """
         self.summary = {}
-        self.dict = set(dictionary)
-        for word in self.dict:
+        self.dictionary = set(dictionary)
```

Comment:
Why do you need `self.dictionary`?

Reply:
Because the function needs initialisation.

Reply:
I don't understand, why cannot you just use `dictionary_` and `summary`?